file/provider/get_volume_profile.go

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright 2025 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package provider ...
+package provider
+
+import (
+	"go.uber.org/zap"
+)
+
+// GetVolumeProfileByName ...
+func (vpcs *VPCSession) GetVolumeProfileByName(name string) error {
```

Comment:
implementation is not aligned with method name, can you implement this as per name, return profile details along with other value so that it can be used with other purpose

Reply:
Done